### DISCUSSION TITLE
Potential bug: images in <noscript> should not be preloaded

### DISCUSSION THREAD

**Discussion by santiagobasulto** [vercel/next.js#58089] [Help]
### Summary

Using Simple Analytics, they have a way to track visits for users that don't have javascript enabled. They do it by using the `<noscript>` tag and requesting an image:

```html
<noscript>
  <img src="https://simpleanalytics.com/noscript.gif>
</noscript>
```

The issue is that Next with SSR will add a preload `img` tag, and duplicate the pageview. Now, I was able to fix this by just using `Image` and disabling preloading.

But I'm wondering, should Next preload images that are contained in a `<noscript>` tag?

**Comment by prod1992** [comment]
I was also facing this same issue. 
I just ended up removing `<noscript>` tags.  

Feels like nowadays it is very unlikely someone to land on your page with disabled javascript. 

Even if that happens, the page might be broken. 
In that scenario - I don't think your pixel is needed :)
